pkgs/applications/editors/emacs/elisp-packages/manual-packages/codeium/default.nix

```
@@ -8,14 +9,24 @@ trivialBuild {
     rev = "1.6.13";
     hash = "sha256-CjT21GhryO8/iM0Uzm/s/I32WqVo4M3tSlHC06iEDXA=";
   };
-
+  commit = "02f9382c925633a19dc928e99b868fd5f6947e58";
   buildInputs = [ pkgs.codeium ];
 
-  patches = [ ./codeium.el.patch ];
-  postPatch = ''
-    substituteInPlace codeium.el --subst-var-by codeium ${pkgs.codeium}/bin/codeium_language_server
+  recipe = writeText "recipe" ''
+    (codeium
+      :repo "Exafunction/codeium.el"
+      :fetcher github
+      :files
+      ("*.el"))
```

Using the default value of `:files` should be enough.
```suggestion
    (codeium
      :repo "Exafunction/codeium.el"
      :fetcher github)
```